### PR TITLE
[AAH-3120] [AAH-2936] Execution Environments > test tab Activity and test Sync from registry

### PR DESCRIPTION
**Pull request by jerabekjiri**
Issue: 
  - [AAH-3120](https://issues.redhat.com/browse/AAH-3120)
  - [AAH-2936](https://issues.redhat.com/browse/AAH-2936)

This PR adds `cy.syncRemoteExecutionEnvironment` cmd to sync `library/alpine` from remote docker registry.

### CODE REVIEW

File: cypress/e2e/hub/execution-environments.cy.ts [hunk 1]
```diff
@@ -204,4 +204,58 @@ describe('Execution Environment Details tab', () => {
...
+
+  it('should successfully sync execution environment from Docker registry', () => {
+    cy.createHubRemoteRegistry().then((remoteRegistry) => {
+      cy.createHubExecutionEnvironment({
+        executionEnvironment: {
+          include_tags: ['latest'],
+          registry: remoteRegistry.id,
+        },
+      }).then((executionEnvironment) => {
+        cy.syncRemoteExecutionEnvironment(executionEnvironment);
```
Comment (nixocio) [inline cypress/e2e/hub/execution-environments.cy.ts:216]: @jerabekjiri we need to delete the created resources as well.  For all those tests.

```suggestion
        cy.syncRemoteExecutionEnvironment(executionEnvironment);
         cy.deleteHubExecutionEnvironment(executionEnvironment).then(() => {
         cy.deleteHubRemoteRegistry(remoteRegistry);
```

**Review by nixocio — CHANGES_REQUESTED** [review]
@jerabekjiri I got the tests locally, and they passed several times. However we could clean up the created resources as well.

File: cypress/e2e/hub/execution-environments.cy.ts (hunk 1 above)
Comment (jerabekjiri) [reply cypress/e2e/hub/execution-environments.cy.ts:216]: Good point, forgot about that :+1:  Thanks

File: cypress/e2e/hub/execution-environments.cy.ts [hunk 2]
```diff
@@ -197,11 +197,72 @@ describe('Execution Environment Details tab', () => {
...
+        cy.deleteHubExecutionEnvironment(executionEnvironment).then(() => {
+          cy.deleteHubRemoteRegistry(remoteRegistry);
+        });
+      });
+    });
+  });
+});
+
+describe('Execution Environment Activity tab', () => {
```
Comment (nixocio) [inline cypress/e2e/hub/execution-environments.cy.ts:225]: @jerabekjiri we still could delete those resources created during those tests here.  Every resource created by a certain test ideally it should be deleted.

**Review by nixocio — APPROVED** [review]

**Pull request merged**